streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDBStore.java

```diff
@@ -178,15 +178,15 @@ void openDB(final Map<String, Object> configs, final File stateDir) {
 
         // Setup statistics before the database is opened, otherwise the statistics are not updated
         // with the measurements from Rocks DB
-        maybeSetUpStatistics(configs);
+        setStatisticsIfNeeded(configs);
 
         openRocksDB(dbOptions, columnFamilyOptions);
         open = true;
 
         addValueProvidersToMetricsRecorder();
     }
 
-    private void maybeSetUpStatistics(final Map<String, Object> configs) {
+    private void setStatisticsIfNeeded(final Map<String, Object> configs) {
```

Comment:
I don't see why this change is required.

Reply:
I saw that the "IfNeeded" convention is used in a lot of places. I can revert it

Reply:
`maybeDoSomething()` is also used in a lot of places. So I agree with @abbccdda.